includes/data.php

```diff
@@ -1497,13 +1498,9 @@ function pods_evaluate_tags( $tags, $sanitize = false ) {
 		return $tags;
 	}
 
-	$callback = 'pods_evaluate_tag';
-
-	if ( true === $sanitize ) {
-		$callback = 'pods_evaluate_tag_sanitized';
-	}
-
-	return preg_replace_callback( '/({@(.*?)})/m', $callback, (string) $tags );
+	return preg_replace_callback( '/({@(.*?)})/m', function ( $tag ) use ( $sanitize, $fallback ) {
```

Comment:
* :no_entry_sign: Opening parenthesis of a multi-line function call must be the last content on the line
* :no_entry_sign: Only one argument is allowed per line in a multi-line function call

<sub>hash: bf5d58b62929713734109281b0896c26</sub>